### PR TITLE
Update help texts (contextual and role) for the Internal role

### LINKED ISSUE #7938: Improve contextual help text for public link role "Internal"
As discussed with @kulmann and @micbar (see https://github.com/owncloud/ocis/issues/5000), the help text describing the role `Internal` needs improvement as users get confused when reading this.

**Current contextual help text:**

**Suggestion:**
_Account and existing access rights required_
The recipient of this link can access the shared content only if he has already an account with at least viewing permissions to that shared content. This role is beneficial to point an existing member to data directly he already has access to, therefore the name Internal is used.

**Current list box text:**

**Suggestion:**
_Internal_
Account and existing access rights required

**NOTE**
The texts are suggestions and can be used for further refinement.

### PR DESCRIPTION
**Pull request by mmattel**
## Description
Improves help texts for the `Internal` role.

## Related Issue
- Fixes #7938

## Motivation and Context
Make the role clear to readers

## How Has This Been Tested?
Text only

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by mmattel** [comment]
Maybe someone can help, the linter complains but I just have exchanged text...?

**Comment by kulmann** [comment]
> Maybe someone can help, the linter complains - but I just have replaced text...?

You can probably run `pnpm lint` to fix it. If it can't be fixed automatically, there will be errors displayed which you can fix manually.

File: packages/web-app-files/src/helpers/contextualHelpers.js [hunk 1]
```diff
@@ -66,12 +66,12 @@ export const shareViaLinkHelp = {
...
   list: [
     {
-      text: $gettext('Only invited people can access'),
+      text: $gettext('Account and existing access rights required'),
       headline: true
     },
     {
       text: $gettext(
-        'Account and login is required. Only people from the list "Invited people" can access.'
+        'The recipient of this link can access the shared content only if he has already an account with at least viewing permissions to that shared content. This role is beneficial to point an existing member to data directly he already has access to, therefore the name Internal is used.'
```
Comment (kulmann) [inline packages/web-app-files/src/helpers/contextualHelpers.js:74]: I don't agree that this text makes it easier to understand. I don't have better ideas... maybe @tbsbdr can have a look.

**Review by kulmann — CHANGES_REQUESTED** [review]

**Comment by mmattel** [comment]
I fixed the linter complains with `pnpm lint --fix`

**Comment by kulmann** [comment]
don't see that this PR is going anywhere at the moment. closing as stalled (trying to clean up the repo). please come up with a new PR when there's agreement in the issue.

**Pull request closed without merging**